README.md

```diff
@@ -46,45 +46,27 @@ It’s a Rails app that:
 
 5. Clone this repo
 
-6. If you don’t have the `bundler` Ruby gem, install it:
+6. Run:
 
     ```sh
-    $ gem install bundler
+    $ bin/setup
     ```
 
-7. Wherever you cloned the repo, go to that directory and install dependencies:
+    This will set up and populate a database with sample data and an admin user. (Read the output for details!)
 
-    ```sh
-    $ bundle install --without production
-    ```
-
-8. Copy the .env.example file to .env - this allows for easy configuration locally.
-
-    ```sh
-    $ cp .env.example .env
-    ```
-
-9. Set up your database. The simple way to do this is:
-
-    ```sh
-    $ bundle exec rake db:setup
-    ```
-
-    That will create a database, set up all the tables, create an admin user, and add some sample data. Make note of the admin user e-mail and password that are shown; you’ll need them to log in and create more users, import more data, or make annotations.
-
-    If you’d like to do the setup manually see [manual postgres setup](#manual-postgres-setup) below.
+    <sub>Alternatively, for manual postgres setup, [see below](#manual-postgres-setup).</sub>
 
     If you're getting error such as `FATAL: role "user" doesn't exist. Couldn't create database.` check [troubleshooting](#troubleshooting) below.
 
-10. Start the server!
+7. Start the server!
 
     ```sh
-    $ bundle exec rails server
+    $ bin/rails server
```

Comment:
Technically, this should still `bundle exec rails server` or `bundle exec bin/rails server`. A lot of the time, skipping the `bundle exec` part will work fine, but if the app is not compatible with a newer version of a gem you might have (e.g. because you are running multiple apps using the same version of Ruby or maybe different branches use different gems), you can have problems. `bundle exec` is what makes sure the right versions of each gem get loaded.

Reply:
`config/boot.rb` loaded within the default `bin/setup` takes care of this i believe:
https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/config/boot.rb

As far as I can tell, `bin/setup` is meant to be all that's needed

Example of another big project that just uses `bin/setup`:
https://github.com/18F/identity-idp/blob/master/bin/setup

(note that within `bin/rake` is where the boot part is loaded)

Reply:
> `config/boot.rb` loaded within the default `bin/setup` takes care of this i believe:

`bundle exec` does actually [set several things about the ruby interpreter](https://github.com/bundler/bundler/blob/master/lib/bundler/shared_helpers.rb#L88-L93) before doing `require 'bundler/setup'` and that simply requiring `bundler/setup` can’t do. But yeah, *mostly* it’s paths, and you don’t need those if you are executing a specific file (`bin/setup`) instead of a command (`setup`), so it doesn’t matter 99% of the time. I guess it’s fine to not do it; most people won’t have a setup that causes a problem without it.

> Example of another big project that just uses `bin/setup`

I didn’t say we shouldn’t use it! BUT, that setup is also monumentally more complex than the 4 commands you are skipping by using the setup file here. They have a really good reason for scripting it. This project is stupidly tiny and simple.